docs/user_guide.md

```diff
@@ -378,6 +378,22 @@ Status:
     Update Time:     2020-04-03T10:04:50+09:00
 ```
 
+### Monitoring with Prometheus
+
+Flink cluster can be monitored with Prometheus in various ways. Here, we introduce the method using podMonitor
+custom resource of Prometheus operator. First, create a Flink cluster the metric exporter activated and its port exposed.
```

Comment:
Missing a "with" between "Flink cluster" and "the metric exporter"?

Reply:
Done.